app/src/main/java/me/toptas/fancyshowcasesample/MainActivity.kt

```diff
@@ -223,28 +227,33 @@ class MainActivity : BaseActivity() {
 
         //Shows a FancyShowCaseView view custom view inflation
         btn_custom_view.setOnClickListener {
-            mFancyShowCaseView = FancyShowCaseView.Builder(this)
-                    .focusOn(it)
-                    .customView(R.layout.layout_my_custom_view, object : OnViewInflateListener {
-                        override fun onViewInflated(view: View) {
-                            view.findViewById<View>(R.id.btn_action_1).setOnClickListener(mClickListener)
-                        }
-                    })
-                    .closeOnTouch(false)
-                    .build()
-            mFancyShowCaseView.show()
+            if (mFancyShowCaseView == null) {
+                mFancyShowCaseView = FancyShowCaseView.Builder(this)
+                        .focusOn(it)
+                        .enableTouchOnFocusedView(true)
+                        .customView(R.layout.layout_my_custom_view, object : OnViewInflateListener {
+                            override fun onViewInflated(view: View) {
+                                view.findViewById<View>(R.id.btn_action_1).setOnClickListener(mClickListener)
+                            }
+                        })
+                        .closeOnTouch(false)
+                        .build()
+                mFancyShowCaseView?.show()
+            }
```

Comment:
There should be the else case. It is not shown if `mFancyShowCaseView` is not null. I will fix it.